### DISCUSSION TITLE
Best way to avoid committing certain files

### DISCUSSION THREAD

**Discussion by aleks-ivanov** [community/community#23220] [New to GitHub]
<p>I am working on a public open source project, to which I commit several times a day.</p>
<p>I have certain extensions in my VS Code, that the owner of the project doesn’t have, but those extensions create files. I don’t want to commit those files, but at the same time, I don’t want to add things to .gitignore and commit a new version of it, because they have nothing to do with the project as a whole.</p>
<p>What would be the best practice to avoid committing such files, without .gitignore and still eliminating human error ?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You can use the <code>.git/info/exclude</code> file. It works just like <code>.gitignore</code>, but limited to the current local repository.</p>

**Comment by aleks-ivanov** [comment]
<p>Thank you, exactly what I needed! :slight_smile:</p>
